### ISSUE TITLE
Adjust target selection based on `mirrord verify-config` output

### ISSUE THREAD

**Issue by Razz4780** [metalbear-co/mirrord-intellij#209]
Blocked by https://github.com/metalbear-co/mirrord/issues/2096

**Comment by eyalb181** [comment]
@Razz4780 Was this done already?

**Comment by aviramha** [comment]
@Razz4780 I believe this was done?

**Comment by Razz4780** [comment]
Nope, doesn't look like it. Now that I look at it, it seems that both this issue and https://github.com/metalbear-co/mirrord-vscode/issues/86 can be replaced with one for the CLI (print only targets compatible with the config)
